source/containers.tex

```diff
@@ -16683,7 +16683,7 @@
 If the set already contains an element equivalent to \tcode{x},
 \tcode{*this} and \tcode{x} are unchanged.
 Otherwise,
-inserts a new element as if by: \tcode{return emplace(std::forward<K>(x));}.
+inserts a new element as if by: \tcode{emplace(std::forward<K>(x));}.
```

Comment:
We conventionally do not include semicolons in expressions, or offset expressions from prose with `:`:
```suggestion
inserts a new element as if by \tcode{emplace(std::forward<K>(x))}.
```